### PR TITLE
Add License

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

**Review by IsaackMwenda — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+SOFTWARE.
+
+
+3RD PARTY EXCEPTIONS
+
+The geometry data for Somalia is a redistribution of a dataset from OCHA Somalia,
+with features and properties modified by Africa's Voices Foundation.
+Those files, along with their source and licence, are:
+
+- `geojson/somalia_regions.geojson`: Adapted from data produced by OCHA Somalia.
```
Comment (IsaackMwenda) [inline LICENSE]: update to Kenya once you update the geojson files

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Retrieved from https://data.humdata.org/dataset/somalia-administrative-boundaries
+January 2020. Licence "Other: humanitarian use only"
+
+- `geojson/somalia_districts.geojson`: Adapted from data produced by OCHA Somalia.
```
Comment (IsaackMwenda) [inline LICENSE]: ditto

**Review by IsaackMwenda — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (as2388) [reply LICENSE]: Good point - let me update the geojson in the #1 too so we're not carrying the somalia geojson in this repo.

Comment (as2388) [reply LICENSE]: updated here using the standard license text for kenyan shapefiles from Project-COVID19-Ke-Urban

**Pull request merged**